programs/wrapper/src/processors/batch_upate.rs

```diff
@@ -48,7 +48,7 @@ pub struct WrapperPlaceOrderParams {
     is_bid: bool,
     last_valid_slot: u32,
     order_type: OrderType,
-    min_out_atoms: u64,
+    min_out_atoms: u64, // TODO: is this still needed?
```

Comment:
plan was to support FillOrKill with a min out amount as part of the fill or kill.

Reply:
but does the trading wrapper still need fill or kill?

Reply:
did not implement, fine with never doing it if you agree